src/components/ec-currency-input/ec-currency-input.vue

```diff
@@ -186,70 +186,59 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/tools/index';
-@import '../../scss/settings/index';
-
-$ec-currency-input-invalid-color: $color-error !default;
-
+<style>
 .ec-currency-input {
-  z-index: 0; // When in currencies dropdown is focused gets a high z-index. We set the index here of the whole component to 0 so we eliminate the chance of competing with other components
+  z-index: 0; /* When in currencies dropdown is focused gets a high z-index. We set the index here of the whole component to 0 so we eliminate the chance of competing with other components */
 
   &__input-group {
-    display: flex;
-    flex-direction: row;
+    @apply tw-flex tw-flex-row;
   }
 
   &__currencies {
-    $currencies-width: 104px;
+    width: var(--currencies-width);
```

Comment:
Don't forget to update references of `--currencies-width` to `--ec-currency-input-currencies-width`.